### PR TITLE
Structure improvements

### LINKED ISSUE #10: Improve code structure
The code should be easy to ready and have single source of truth.

### PR DESCRIPTION
**Pull request by melnelen**
Resolves #10

### CODE REVIEW

File: Shamaz.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -12,7 +12,10 @@
 		D02ACE18242808BA003938D1 /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = D02ACE17242808BA003938D1 /* ViewController.swift */; };
 		D02ACE1B242808BA003938D1 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = D02ACE19242808BA003938D1 /* Main.storyboard */; };
 		D02ACE1D242808BC003938D1 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = D02ACE1C242808BC003938D1 /* Assets.xcassets */; };
-		D02ACE20242808BC003938D1 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = D02ACE1E242808BC003938D1 /* LaunchScreen.storyboard */; };
+		D02D1AAB2431579900DE9B6F /* phrases.swift in Sources */ = {isa = PBXBuildFile; fileRef = D02D1AAA2431579900DE9B6F /* phrases.swift */; };
```
Comment (cyril-lepottier) [inline Shamaz.xcodeproj/project.pbxproj]: Any reason for the file name to start with a lower case?
This comment also apply for  `design.swift`

File: Shamaz/ViewController.swift [hunk 2]
```diff
@@ -8,72 +8,107 @@
 
 import UIKit
 
+@available(iOS 13.0, *)
```
Comment (cyril-lepottier) [inline Shamaz/ViewController.swift]: Won't it prevent your app from running on iOS 11?

File: Shamaz/SecondViewController.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+import UIKit
+
+@available(iOS 13.0, *)
+class SecondViewController: UIViewController {
+
+    @IBOutlet weak var nextPlayerButton: UIButton!
+    @IBOutlet weak var playerTaskLabel: UILabel!
+    
+    var recievedPhrase = "Player's task"
```
Comment (cyril-lepottier) [inline Shamaz/SecondViewController.swift]: Typo: `receivedPhrase`

File: Shamaz/SecondViewController.swift [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    var nextPlayerPhrase = ""
+    var nextPlayerNumber = Int.random(in: 1...10)
+    
+    let design = Design()
```
Comment (cyril-lepottier) [inline Shamaz/SecondViewController.swift]: Alternatively, you could use `static` or `Singleton`

File: Shamaz/SecondViewController.swift [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+//  Shamaz
+//
+//  Created by Alexandra Ivanova on 31/03/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+@available(iOS 13.0, *)
+class SecondViewController: UIViewController {
```
Comment (cyril-lepottier) [inline Shamaz/SecondViewController.swift]: Now you have 2 view controllers, you should rename them so the reader can understand what they do.

File: Shamaz/design.swift [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import UIKit
+
+struct Design {
+    let textColor = UIColor(white: 1.00, alpha: 1.00)
+    let labelFont = UIFont(name: "Chalkduster", size: 30.0)
+    let buttonFont = UIFont(name: "Chalkduster", size: 20.0)
+    let buttonRadius:CGFloat = 5.00
+    let lightGreenColor = UIColor(red:0.56, green:1.00, blue:0.66, alpha:1.00)
+    let lightBlueColor = UIColor(red:0.45, green:1.00, blue:0.97, alpha:1.00)
+    let midnightBlue = UIColor(red:0.13, green:0.16, blue:0.22, alpha:1.00)
```
Comment (cyril-lepottier) [inline Shamaz/design.swift]: Pro tip: In big applications, using theses names can create issues. I have 2 scenarios in mind:
1. Rebranding of the app, suddenly, you have to replace `lightGreenColor` by `pinkColor` everywhere in your app.
2. Dark mode, for dark mode user `lightGreenColor` should actually display `darkGreenColor`.

**Review by cyril-lepottier — CHANGES_REQUESTED** [review]
2 points mainly:
- I think further improvements can be done in the structure of your project.
- The app won't run on iOS 11, I think

**Comment by cyril-lepottier** [comment]
The base branch of this pull request is badly set, all its commit are also available in #10

File: Shamaz.xcodeproj/project.pbxproj (hunk 1 above)
Comment (melnelen) [reply Shamaz.xcodeproj/project.pbxproj]: No particular reason. I had a problem renaming it with a capital letter after creating it so I had to rename it to phrases2 and then -> Phrases. Weird XCode behavior!

File: Shamaz/ViewController.swift (hunk 2 above)
Comment (melnelen) [reply Shamaz/ViewController.swift]: Yes, I was not sure how to setup iOS11. Is it really necessary?

File: Shamaz/SecondViewController.swift (hunk 3 above)
Comment (melnelen) [reply Shamaz/SecondViewController.swift]: fixed :)

File: Shamaz/SecondViewController.swift (hunk 4 above)
Comment (melnelen) [reply Shamaz/SecondViewController.swift]: I'll use static. Thnx :)

File: Shamaz/Controllers/GameStartViewController.swift [hunk 7]
```diff
@@ -8,49 +8,41 @@
 
 import UIKit
 
-@available(iOS 13.0, *)
-class ViewController: UIViewController {
+class GameSatrtViewController: UIViewController {
```
Comment (cyril-lepottier) [inline Shamaz/Controllers/GameStartViewController.swift]: Typo

File: Shamaz/Controllers/PlayersTaskViewController.swift [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    var nextPlayerPhrase = ""
+    var nextPlayerNumber = Int.random(in: 1...10)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        nextPlayerButton.layer.cornerRadius = Design.buttonRadius
+        nextPlayerButton.backgroundColor = .white
+        nextPlayerButton.setTitleColor(Design.primaryThemeColor, for: [])
+        nextPlayerButton.setTitle("Who's NeXT?", for: [])
```
Comment (cyril-lepottier) [inline Shamaz/Controllers/PlayersTaskViewController.swift]: Typo, Maybe?

**Review by cyril-lepottier — APPROVED** [review]

File: Shamaz/Controllers/PlayersTaskViewController.swift (hunk 8 above)
Comment (melnelen) [reply Shamaz/Controllers/PlayersTaskViewController.swift]: Who's NeXT?
I got that from the project description actually: "Add a third button for the next player nomination: Ex."Who's NeXT?""
😜

File: Shamaz/Controllers/GameStartViewController.swift (hunk 7 above)
Comment (melnelen) [reply Shamaz/Controllers/GameStartViewController.swift]: yes 😊

File: Shamaz/Controllers/PlayersTaskViewController.swift (hunk 8 above)
Comment (cyril-lepottier) [reply Shamaz/Controllers/PlayersTaskViewController.swift]: Ok 😊

**Pull request merged**